### ISSUE TITLE
States as an :atom

### ISSUE THREAD

**Issue by xantrac** [subvisual/fsmx#11]
Hey team, thank you for the work you are doing here.

Quick question, any specific reason why atoms are not valid states in the type spec?
![image](https://user-images.githubusercontent.com/38252032/202016850-dfc5d406-f80b-4716-9b5b-8c2e425a9068.png)

This works just fine but I'm getting mismatching type in the dialyzer
```
  def submit_order(order, order_params) do
    {:ok, %{new: new}} =
      Multi.new()
      |> Fsmx.transition_multi(order, :new, :submitted, order_params)
      |> Repo.transaction()

    {:ok, Repo.reload(new)}
  end
```

**Comment by naps62** [comment]
Hi @xantrac 
I believe this has been fixed in #3 , but never actually release to the package manager

I'm no longer actively working on Elixir, so that's why I ended up forgetting that when merging those fixes
In the meantime, you can use the direct git repo as a dependency, and it should fix your issue

**Comment by naps62** [comment]
@xantrac I just pushed a new release. apologies for the huge delay
